goss-testing/scripts/check_bios_firmware_versions.sh

```diff
@@ -57,8 +57,37 @@ function set_vars() {
   fi
 
   # Allow for output of documentation when needed
+  rc=0
+
+  DL325_FW="2.46"
+  DL385_FW="2.46"
+
+  DL325_BIOS="A43"
+  DL385_BIOS="A42"
+
+  # Taken from HFP-firmware-2.0.111516-0 but leaving here for reference in this code
+  # Appropriate versions to use for Shasta 1.5.0 or HPCM as of 11/4/2021 unless otherwise indicated by your
+  # software installation manual:
```

Comment:
I don't believe that is the correct version of HFP for the Raspberry release (CSM 1.2) per: https://connect.us.cray.com/confluence/display/HPCSWPM/Raspberry+Recipe which is HFP 22.05. Shasta 1.5 contains CSM 1.0, not CSM 1.2 🙂

The BIOS version might not be correct, as this is what I pulled out of the HFP tarball that is supposed to go with raspberry :
```
$ cat HFP-firmware-22.05.0/NOTES.txt | grep HPE_DL
    HPE_DL325_A43                       BIOS 2.52, HPE_ILO5 version 2.46
    HPE_DL385_A42                       BIOS 2.52, HPE_ILO5 version 2.46
```

Reply:
Got it.  I can update that here and just adjust the version in variable

Reply:
Fixed.